frontend/javascripts/oxalis/model/accessors/dataset_accessor.js

```diff
@@ -612,7 +631,10 @@ function _getUnrenderableLayersForCurrentZoom(state: OxalisState) {
         return resolutionInfo.hasIndex(fallbackZoomStep);
       });
     })
-    .map<DataLayerType>(({ layer }) => layer);
+    .map<[DataLayerType, SmallerOrHigherInfo]>(({ layer, resolutionInfo }) => {
+      const hasSmallOrHigher = resolutionInfo.hasSmallerAndOrHigherIndex(zoomStep);
+      return [layer, hasSmallOrHigher];
```

Comment:
I think, I would prefer an object as a return value, since this is often easier to read/use (instead of `value[0]` it would be `value.layer` for example)
```suggestion
      return { layer, hasSmallOrHigher };
```

Reply:
please also rename `_getUnrenderableLayersForCurrentZoom` to something like `_getUnrenderableLayerInfosForCurrentZoom`. also adapt the functions comment to the new return type. ideally, also add the flow return signature.